lib/providers/restful-provider.js

```diff
@@ -0,0 +1 @@
+module.exports = function(){};
```

Comment:
Yeah I wasn't paying attention to which file this was in. I was thinking it was a function accessible to the user because it's in the SDK, in which case i think it should throw an error. I take it back now though.